plugin/lsp.vim

```diff
@@ -28,6 +28,7 @@ let g:lsp_highlight_references_enabled = get(g:, 'lsp_highlight_references_enabl
 let g:lsp_preview_float = get(g:, 'lsp_preview_float', 1)
 let g:lsp_preview_autoclose = get(g:, 'lsp_preview_autoclose', 1)
 let g:lsp_preview_doubletap = get(g:, 'lsp_preview_doubletap', [function('lsp#ui#vim#output#focuspreview')])
+let g:lsp_popup_max_width = get(g:, 'lsp_popup_max_width', -1)
```

Comment:
We should either call both float or popup (meaning `g:lsp_preview_float` and `g:lsp_popup_max_width`).